db/projects/bluzelle/asset-blz-logo-square.svg

```diff
@@ -0,0 +1 @@
0.7s-18.2 40.7-40.7 40.7-40.7-18.2-40.7-40.7C-100.2 16.3-82-2-59.6-2z"/></g><g transform="rotate(-45 208.877 -442.426)" id="Subtract"><path id="path1_fill" class="st0" d="M-384.6 22v60.7h55.6c-4-9.3-6.2-19.5-6.2-30.4 0-10.7 2.2-21 6.2-30.4l-55.6.1z"/></g><g transform="rotate(-45 226.004 -449.52)" id="Subtract_1_"><path id="path2_fill" class="st0" d="M-174 70h55.6V9.3H-174c4 9.3 6.2 19.5 6.2 30.4 0 10.8-2.2 21-6.2 30.3z"/></g></g><g id="Group_1_"><g transform="rotate(45 184.016 477.075)" id="Ellipse_2_"><path id="path0_fill_2_" class="st1" d="M236 305.4c-22.5 0-40.7-18.2-40.7-40.7 0-22.5 18.2-40.7 40.7-40.7s40.7 18.2 40.7 40.7c0 22.4-18.3 40.7-40.7 40.7z"/></g><g transform="rotate(45 152.468 464.008)" id="Ellipse_3_"><path id="path0_fill_3_" class="st1" d="M-144.6 284.2c-22.5 0-40.7-18.2-40.7-40.7 0-22.5 18.2-40.7 40.7-40.7 22.5 0 40.7 18.2 40.7 40.7 0 22.5-18.2 40.7-40.7 40.7z"/></g><g transform="rotate(45 175.31 474.522)" id="Subtract_2_"><path id="path3_fill" class="st1" d="M175.8 293.5v-60.7h-55.6c4 9.3 6.2 19.5 6.2 30.4 0 10.7-2.2 21-6.2 30.4l55.6-.1z"/></g><g transform="rotate(45 158.183 467.428)" id="Subtract_3_"><path id="path4_fill" class="st1" d="M-30.6 215.1h-55.6v60.7h55.6c-4-9.3-6.2-19.5-6.2-30.4 0-10.7 2.2-21 6.2-30.3z"/></g></g><g id="Group_2_"><g transform="translate(379.068 7)" id="Ellipse_4_"><path id="path0_fill_4_" class="st2" d="M-104.1 33.7c0 22.5-18.2 40.7-40.7 40.7s-40.7-18.2-40.7-40.7S-167.3-7-144.8-7s40.7 18.2 40.7 40.7z"/></g><g transform="translate(379.068 33.136)" id="Ellipse_5_"><path id="path0_fill_5_" class="st2" d="M-104.1 396.3c0 22.5-18.2 40.7-40.7 40.7s-40.7-18.2-40.7-40.7 18.2-40.7 40.7-40.7 40.7 18.2 40.7 40.7z"/></g><g transform="translate(381.076 13.903)" id="Subtract_4_"><path id="path5_fill" class="st2" d="M-132 88.8h-25.6v50.4c4.2-.7 8.4-1.1 12.8-1.1s8.6.4 12.8 1.1V88.8z"/></g><g transform="translate(381.076 28.465)" id="Subtract_5_"><path id="path6_fill" class="st2" d="M-132 290.8c-4.2.7-8.4 1.1-12.8 1.1s-8.6-.4-12.8-1.1v50.4h25.6v-50.4z"/></g></g><g id="Group_3_"><g transform="rotate(90 188.966 209.034)" id="Ellipse_6_"><path id="path0_fill_6_" class="st3" d="M244.6-16.3c0 22.5-18.2 40.7-40.7 40.7s-40.7-18.2-40.7-40.7S181.4-57 203.9-57s40.7 18.2 40.7 40.7z"/></g><g transform="rotate(90 175.898 195.966)" id="Ellipse_7_"><path id="path0_fill_7_" class="st3" d="M251.5 331.2c0 22.5-18.2 40.7-40.7 40.7s-40.7-18.2-40.7-40.7 18.2-40.7 40.7-40.7 40.7 18.2 40.7 40.7z"/></g><g transform="rotate(90 184.51 206.587)" id="Subtract_6_"><path id="path7_fill" class="st3" d="M221.6 80.3V29.9H196v50.4c4.2-.7 8.4-1.1 12.8-1.1s8.6.4 12.8 1.1z"/></g><g transform="rotate(90 177.23 199.306)" id="Subtract_7_"><path id="path8_fill" class="st3" d="M221.6 222.3c-4.2.7-8.4 1.1-12.8 1.1s-8.6-.4-12.8-1.1v50.4h25.6v-50.4z"/></g></g><g transform="translate(377.844 18.844)" id="Ellipse_8_"><path id="path9_fill" class="st4" d="M-87.1 215c0 31.8-25.8 57.7-57.7 57.7s-57.7-25.8-57.7-57.7 25.8-57.7 57.7-57.7 57.7 25.9 57.7 57.7z"/></g></svg>
```

Comment:
I am not sure, but I would say that their official logo is this:
https://bluzelle.com/assets/img/Bluzelle%20-%20Screen%20-%20Symbol%20-%20Big%20-%20Blue%20cut.png
source: https://bluzelle.com/

Reply:
As well as their full logo:
https://bluzelle.com/assets/img/Bluzelle%20-%20Screen%20-%20Logo%20-%20Big%20-%20Blue.png